templates/default/dotnet/setup.config.erb

```diff
@@ -17,6 +17,15 @@
       <IIS>
         <automatic enabled="true" />
       </IIS>
+	  <% unless @standalone_apps.nil? %> 
```

Comment:
Indentation in this block is very strange. Looks like the rest of the file is two spaces, so we should keep it as two consistently.